ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py

```diff
@@ -249,3 +249,32 @@ def test_paddle_divide(
         x=x[0],
         y=x[1],
     )
+
+
+# add
+@handle_frontend_test(
+    fn_tree="paddle.add",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        shared_dtype=True,
```

Comment:
@MahmoudAshraf97 Should we use a shared dtype here? Should tensor math functions handle different dtypes?